miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/userinfo/UserInfoHandler.kt

```diff
@@ -0,0 +1,76 @@
+package com.rakuten.tech.mobile.miniapp.userinfo
+
+import com.rakuten.tech.mobile.miniapp.js.CallbackObj
+import com.rakuten.tech.mobile.miniapp.js.MiniAppMessageBridge
+import com.rakuten.tech.mobile.miniapp.permission.MiniAppCustomPermissionResult
+import com.rakuten.tech.mobile.miniapp.permission.MiniAppCustomPermissionType
+
+@Suppress("TooGenericExceptionCaught", "LongMethod")
+internal class UserInfoHandler(private val bridge: MiniAppMessageBridge) {
```

Comment:
There is one thing I missed. We should pass only the listener of `MiniAppMessageBridge` to execute `postValue`, `postError` and `MiniAppCustomPermissionCache`.

```suggestion
internal class UserInfoHandler(
private val bridgeListener: MAMessageBridgeListener,
private val customPermissionCache: MiniAppCustomPermissionCache) {
```

Reply:
I have updated this part. Now, we need to make sure that miniapp has been created before initializing `UserInfoHandler` as `MiniAppCustomPermissionCache` is an internal class.